rollup.compact.config.js

```diff
@@ -0,0 +1,10 @@
+import npm from 'rollup-plugin-npm';
+import baseOptions from './rollup.base';
+
+export default {
+    plugins: [
+        npm(Object.assign({}, baseOptions, {
+          compact: true,
```

Comment:
It would be preferable to use the same minifier for all of this library's distributions, namely Closure Compiler, as we do here: https://github.com/mixpanel/mixpanel-js/blob/f0fed1b541acc9d6cd2fac2234a9b5da3646dd21/build.sh#L14